.build-tools/pkg/metadataanalyzer/utils.go

```diff
@@ -68,17 +81,17 @@ func GenerateMetadataAnalyzer(contribRoot string, componentFolders []string, out
 
 		switch componentType {
 		// Only the component types listed here implement the GetComponentMetadata method today
-		case "secretstores":
+		case string(mdutils.SecretStoreType):
```

Comment:
Please note these are the folder names here and not necessarily the component names!!

They are intentionally distinct from what you found in mdutils because the component category folder name might not match the component type exactly.